### DISCUSSION TITLE
Show 'watched' status for watched videos

### DISCUSSION THREAD

**Discussion by stackingflow** [ReVanced/revanced-discussions#131] [Question]
Issue
- Sometimes the videos we have already watched are recommended to us again in our home feed. 
- Its hard to distinguished the video we have already watched.

Solution
Show a `watched` tag (text inside a box) on the top left of the video.

**Comment by oSumAtrIX** [comment]
Would require a local database, but nice idea

**Reply by TheJeterLP** [reply]
Wouldn't appear as watched if I watch a video on another device/client, so in my opinion not possible to do.

**Comment by LisoUseInAIKyrios** [comment]
YT now shows this as a seekbar in the thumbnail.
